src/apps/HeyGPT/HeyGPT/Helpers/AppWindowHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using HeyGPT.App.Views;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace HeyGPT.App.Helpers;
+
+// TODO: We really should have a service that manages all the application's windows
```

Comment:
This is a temporary helper class... We really should have a proper WindowManagementService to handle all the fun windows related stuff.    Experience says that there is a lot we need to manage.